test/com/google/enterprise/adaptor/sharepoint/SharePointAdaptorTest.java

```diff
@@ -1507,6 +1503,12 @@ public void testGetDocContentSCVerifyGroupPush() throws Exception {
       tmp.put(SITES_SITECOLLECTION_VISITORS, Arrays.<Principal>asList());
       goldenGroups2 = Collections.unmodifiableMap(tmp);
     }
+    ContentExchange response1 = new ContentExchange(
```

Comment:
Add a comment in both tests: "The first call to <appropriate SOAP method> returns 2 groups, and the second call returns 3 groups." (reversed for the delete test, naturally).

Reply:
Done